roles/kubespray-defaults/defaults/main/download.yml

```diff
@@ -178,6 +185,7 @@ containerd_download_url: "https://github.com/containerd/containerd/releases/down
 cri_dockerd_download_url: "https://github.com/Mirantis/cri-dockerd/releases/download/v{{ cri_dockerd_version }}/cri-dockerd-{{ cri_dockerd_version }}.{{ image_arch }}.tgz"
 skopeo_download_url: "https://github.com/lework/skopeo-binary/releases/download/{{ skopeo_version }}/skopeo-linux-{{ image_arch }}"
 yq_download_url: "https://github.com/mikefarah/yq/releases/download/{{ yq_version }}/yq_linux_{{ image_arch }}"
+scheduler_plugins_src_download_url: "https://github.com/kubernetes-sigs/scheduler-plugins/archive/{{ scheduler_plugins_version }}.tar.gz"
```

Comment:
If it copies the manifest yaml as an ansible template, it does not need to download the helm archive tgz.